efficientdet/main.py

```diff
@@ -240,10 +241,16 @@ def _can_partition(spatial_dim):
       per_host_input_for_training=tf.estimator.tpu.InputPipelineConfig
       .PER_HOST_V2)
 
+  if FLAGS.model_dir is None:
```

Comment:
Maybe we can simple mark model_dir as required?

If users get used to specify model_dir=None or not specify it, then they will often have troubles to continue a stopped training job: when they restart the training job, model_dir will change and the job wouldn't start from the previously trained checkpoionts.

WDYT?

Reply:
I specific model dir by shell in my repo(Also use timestamp).
But for beginners, it's very convenient.
And I believe continue training is an advanced feature, they have ability to specific model dir when they need to restart training jobs.
Two strategies are goods for me.